modules/loader-utils/test/lib/request-utils/request-scheduler.spec.js

```diff
@@ -17,5 +17,18 @@ test('RequestScheduler#scheduleRequest', async t => {
   result = await requestScheduler.scheduleRequest({id: 2}, () => -1);
   t.notOk(result);
 
+  let priority = 0;
+  result = await Promise.all([
+    requestScheduler.scheduleRequest({id: 3}).then(() => {
```

Comment:
A comment here explaining the test case would be helpfule. It takes a bit of effort to reconstruct what this is testing, the maxRequests being 1 etc.

This comment is probably applies for the existing code too, maybe I should go back and write some docs for this class.